recipe/build.sh

```diff
@@ -27,20 +95,14 @@ if [[ ${PY_INTERP_LINKAGE_NATURE} == shared ]]; then
   _ENABLE_SHARED=--enable-shared
 fi
 
-# For debugging builds, set this to 0 to disable profile-guided optimization
+# For debugging builds, set this to no to disable profile-guided optimization
```

Comment:
Is this still true or does QUICK_BUILD take precedence

Reply:
Not sure, will get cleaned up.